pkg/cmd/tknpac/bootstrap/bootstrap.go

```diff
@@ -60,6 +61,13 @@ func install(ctx context.Context, run *params.Run, opts *bootstrapOpts) error {
 		// nolint:forbidigo
 		fmt.Println("🏃 Checking if Pipelines as Code is installed.")
 	}
+	tektonInstalled, err := checkPipelinesInstalled(run)
+	if err != nil {
+		return err
+	}
+	if !tektonInstalled {
+		return errors.New("tekton API not found on the cluster. Please install OpenShift Pipelines first")
```

Comment:
we want to make it compatible tekton upstream too, so maybe if we can change the wording to Tekton :)